datajunction-server/datajunction_server/api/main.py

```diff
@@ -63,7 +83,7 @@
         "name": "MIT License",
         "url": "https://mit-license.org/",
     },
-    dependencies=dependencies,
```

Comment:
After being removed from here, I don't see the `dependencies` variable used anywhere in this module anymore. Should we still define it then on line 58?

Reply:
Good point, will remove that